api/azure/webhook.go

```diff
@@ -17,6 +18,15 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
+// Labels for runs from Azure Pipelines are determined from the artifact names.
+// For master runs, artifact name may be either just "results" or something
+// like "safari-results".
+var (
+	masterRegex = regexp.MustCompile(`[^-]results$`)
```

Comment:
`[^-]` is "not a dash", right? so this would match aresults but not safari-preview-results ?

In either case, add a unit test to webhook_test.go that covers the positive cases :)

Reply:
D'oh! Can `processAzureBuild` as a whole be tested, or should I write tests for the regexes?